data/brands/shop/outdoor.json

```diff
@@ -93,6 +93,18 @@
         "shop": "outdoor"
       }
     },
+    {
+      "displayName": "Fjällräven",
+      "id": "fjallraven-d45246",
+      "locationSet": {"include": ["001"]},
```

Comment:
they only seem to have their own stores in the US and Canada everywhere else it's partners

Reply:
i ran a build after changing it so it would be happy, and the build changed some other things, sorry

Reply:
the build looks to be correct and you fixed that small bit fast